Sources/SwiftLexicalLookup/Configurations/LookupConfig.swift

```diff
@@ -13,14 +13,26 @@
 @_spi(Experimental) public struct LookupConfig {
   /// Specifies behavior of file scope.
   @_spi(Experimental) public var fileScopeHandling: FileScopeHandlingConfig
+  /// Specifies whether lookup should finish in the closest sequential scope.
+  @_spi(Experimental) public var finishInSequentialScope: Bool
+  /// Specifies whether to include results generated in file and member block scopes.
+  @_spi(Experimental) public var includeMembers: Bool
```

Comment:
I don’t know if you have examples that do set these to `true` but if you do, I think they would help readers understand what the options are used for.

Reply:
I added some examples to the documentation comments of those new configurations. 

I think `includeMembers` is particularly interesting because if we just consider the compiler, it always excludes members and file scope classes in unqualified lookup. I suppose those are handled by a separate member lookup. 

Should we then consider members in `SwiftLexicalLookup` and allow this sort of „semi-member lookup” (as modeled by this `includeMembers` config) or should we just skip them and don’t provide this functionality of ASTScope unqualified lookup? I think I'd also like to hear @DougGregor opinion on this one too.